### ISSUE TITLE
`x86_64-w64-mingw32-gcc`: `error: process b terminated abnormally: aborted` -  No `assert()`/additional information

### ISSUE THREAD

**Issue by helmesjo** [build2/build2#446]
Under Cygwin64, checkout commit [1340fce77d536c6f](https://github.com/helmesjo/build2-FFmpeg/commit/1340fce77d536c6f20a19223c992a0aecc279a36).

```
$ bdep init -C @mingw cc config.c=x86_64-w64-mingw32-gcc config.c.coptions="-O2"
$ bdep update config.c.coptions="-O2"
# required according to upstream ^
```

The problem appears to be related to headers not being found (`mingw` appears to require `-iquote` to behave according to the standard of finding headers in the same directory when using `#include "header.h"`).

[update.txt](https://github.com/user-attachments/files/17588059/update.txt)

**Note:** This isn't as much about supporting this use-case, rather to fix the crash.

**Comment by boris-kolpackov** [comment]
Looking at the log, I have no idea what's going on or what `-iquote` has to do with it. If you understand what's causing the abort, please add this information to the issue.

**Comment by helmesjo** [comment]
Were you not able to reproduce the crash using the above details?

**Comment by helmesjo** [comment]
Makes sense, I just didn't understand you're request.

**Comment by boris-kolpackov** [comment]
> I just didn't understand your request.

You seem to have some ideas about what may be going on (based on your mentioning of `-iquote`) but you don't provide any details so I have no idea what it is that you may know.

**Comment by helmesjo** [comment]
Yep yep, I didn't understand the first time but got it the second time you clarified. I'll have a look at it.
